webapp/channels/src/components/pricing_modal/card.tsx

```diff
@@ -12,9 +12,9 @@ import ExternalLink from 'components/external_link';
 
 import {HostedCustomerLinks} from 'utils/constants';
 
-import BlankCardImage from './blank_card_image.svg';
 import BuildingSvg from './building.svg';
 import TadaSvg from './tada.svg';
+import ChatIllustration from 'components/threading/common/chat_illustration';
```

Comment:
nit: Seems a bit odd to reference this from the threading components that is not related at all with pricing modal. Could we move the svg to: https://github.com/mattermost/mattermost/tree/master/webapp/channels/src/components/common/svg_images_components